hsmodels/schemas/rdf/resource.py

```diff
@@ -95,7 +92,6 @@ class ResourceMetadataInRDF(BaseModel):
     _parse_description = root_validator(pre=True, allow_reuse=True)(rdf_parse_description)
 
     _parse_identifier = validator("identifier", pre=True, allow_reuse=True)(rdf_parse_identifier)
-    _parse_rdf_sources = validator("sources", pre=True, allow_reuse=True)(parse_rdf_sources)
```

Comment:
The validator function should be removed too since it is no longer used anywhere.

Reply:
Fixed